ground/gcs/src/plugins/config/configautotunewidget.cpp

```diff
@@ -544,15 +581,22 @@ QJsonDocument ConfigAutotuneWidget::getResultsJson()
     pitch_ident["bias"] = m_autotune->measuredPitchBias->text().toDouble();
     pitch_ident["noise"] = m_autotune->measuredPitchNoise->text().toDouble();
     identification["pitch"] = pitch_ident;
+    QJsonObject yaw_ident;
+    yaw_ident["gain"] = m_autotune->measuredYawGain->text().toDouble();
+    yaw_ident["bias"] = m_autotune->measuredYawBias->text().toDouble();
+    yaw_ident["noise"] = m_autotune->measuredYawNoise->text().toDouble();
+    identification["yaw"] = yaw_ident;
+
     identification["tau"] = m_autotune->rollTau->text().toDouble();
     json["identification"] = identification;
 
-    QJsonObject tuning, parameters, computed;
+    QJsonObject tuning, parameters, computed, misc;
     parameters["damping"] = m_autotune->lblDamp->text().toDouble();
-    parameters["noiseSensitivity"] = m_autotune->lblNoise->text().toDouble();
+    parameters["noiseSensitivity"] = m_autotune->lblNoise->text().split(" ").at(0).toDouble();
```

Comment:
I have been bitten by the "at" function before. If for some reason the "split" returns an empty list the "at" will segfault. Although cumbersome the size should be checked before "at" is used.